build/constants.yaml

```diff
@@ -275,7 +275,7 @@
     default-version-file-prefix: defaultVersion
     default-version-file-type: txt
     versions-to-build-file-name: versionsToBuild.txt
-    container-metadata-url-format: '{0}/{1}?restype=container&comp=list&include=metadata'
+    container-metadata-url-format: '{0}/{1}?restype=container&comp=list&include=metadata&marker={2}'
```

Comment:
When changes are made to the constants.yaml file, generally it's good to run the `./build/generateConstants.sh` script. You can see [here](https://github.com/microsoft/Oryx/blob/main/build/constants.yaml#L314-L315) that this change should also be propagating to our go script, and running the script will ensure that it does!

There are some fixes to the constants.yaml file in the current main branch, so make sure to pull down main and merge its changes into your branch before running the script!

Reply:
hmmm yes I would think that we should also see [this line](https://github.com/microsoft/Oryx/blob/main/src/startupscriptgenerator/src/common/consts/sdk_storage_constants.go#L16) have this change. Perhaps it is on your local branch and hasn't been commited?

Reply:
Correct, it was not staged. Thanks for pointing it out!